### PR TITLE
Trying to use last opencv

### PR DESCRIPTION
**Pull request by lock042**
Trying to use last opencv version

### CODE REVIEW

**Comment by hfiguiere** [comment]
There is no opencv4 generated.

File: org.free_astro.siril.json [hunk 1]
```diff
@@ -94,13 +94,14 @@
             ],
             "config-opts": [
                 "-D CMAKE_BUILD_TYPE=RELEASE",
-                "-D BUILD_EXAMPLES=OFF"
+                "-D BUILD_EXAMPLES=OFF",
+                "-D OPENCV_GENERATE_PKGCONFIG=ON"
```
Comment (hfiguiere) [inline org.free_astro.siril.json]: This is the error. It should be `"-DOPENCV_GENERATE_PKGCONFIG=ON"` (note the lack of space after `-D`). Also same for the previous lines.

I'm not sure why CMake doesn't error.

(I tested and the build is successfull)

Comment (hfiguiere) [reply org.free_astro.siril.json]: For the record I'm to blame for the original syntax error. Probably copy-pasta from elsewhere.

Comment (lock042) [reply org.free_astro.siril.json]: No. I'm not a cmake user, I did not know.
I've fixed it, it looks ok now.

**Pull request merged**